README.md

```diff
@@ -2,6 +2,13 @@
 
 A unified publishing application for content on GOV.UK
 
+## Nomenclature
+
+  * Content - Some text (and related fields) a user wants to publish
+  * Revision - A version of a piece of content in a particular locale
+  * Edition - A revision that is currently in a content store (draft/live)
```

Comment:
Should this be a revision that is in the Publishing API? e.g. we have all the past editions in Publishing API

Reply:
Any better now?

Reply:
Yes - though this implies that Publishing API exists on draft and live stack when that happens when you get to Content Store. I'd just drop off the stack part.